doc/man3/SSL_CTX_set_ec_point_formats.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_ec_point_formats,
+SSL_set_ec_point_formats
+- configure the available EC point formats
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_CTX_set_ec_point_formats(SSL_CTX *ctx, const unsigned char *formats,
+                                  unsigned int formats_len);
+ int SSL_set_ec_point_formats(SSL *ssl, const unsigned char *formats,
```

Comment:
Probably we should call both of the these functions `set1` rather than just `set` since the memory ownership does not transfer.